packages/commonwealth/client/scripts/views/pages/search/index.tsx

```diff
@@ -57,6 +57,25 @@ const SearchPage = () => {
   const [urlQueryParams] = useSearchParams();
   const [bottomRef, bottomInView] = useInView();
 
+  //TODO revisit to add hook
```

Comment:
I have a question out to @jessmart1213 on how to handle this functionality, there is a hook made `useWindowResize` but this may not even be needed if breadcrumbs won't exist on this niche page. It's not currently on the figma's sitemap.

Reply:
@jessmart1213 And I spoke about the above and we are going to leave it as-is for now. Design and product are going to revamp the search page and that will eventually surface more clarity on what this will look like.